oeps/processes/oep-0057-core-repositories.rst

```diff
@@ -0,0 +1,166 @@
+===========================
+OEP-57: Core Repositories
+===========================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP-57
+     - :doc:`OEP-57 <oep-0057-core-repositories>`
```

Comment:
Nit: According to the OEP spec, you should be putting `proc` in the document name (`oep-0057-proc-core-repositories`, see https://github.com/openedx/open-edx-proposals/tree/master/oeps/processes - all (except oep-1 lolol, and oep-54 where I also missed the notice) and https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#oep-header-preamble)